firmware/examples/Temperature/source/main.cpp

```diff
@@ -0,0 +1,25 @@
+#include "L2_HAL/sensors/temperature/temperature.hpp"
+#include "utility/log.hpp"
+
+int main(void)
+{
+  Temperature temperature_example;
+  bool temperature_sensor_initialized = temperature_example.Initialize();
+  LOG_INFO("Temperature Device set: %d \n", temperature_sensor_initialized);
+
+  if (temperature_sensor_initialized)
```

Comment:
Can you make this a loop so users can see the temperature data changing in realtime?